### ISSUE TITLE
Cant find recurvebow and longbow in primitive plus drops item lists

### ISSUE THREAD

**Issue by Ryah535** [thommcgrath/Beacon#107]
I have noticed a few things missing from the item lists for primitive plus. Would love to see an update for primitive plus. Also there are few outdated items that are no longer in the game.

Thanks!

**Comment by thommcgrath** [comment]
I didn’t know Prim+ was even updated anymore. Do you have a list of changes?

**Comment by Ryah535** [comment]
Um it might take me a bit. Do you need just the names of the items or the
code entries? Item names would be quick.

**Comment by thommcgrath** [comment]
As long as I can find the spawn codes somewhere, I can do that.

**Comment by Ryah535** [comment]
Alrighty thanks for such a quick response. Ill get ya a list asap. :)

**Comment by Ryah535** [comment]
Okay here ya go. This is what I know off the bat.
ADD:
LongBow-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/Weapons/PrimalItem_WeaponLongBow.PrimalItem_WeaponLongBow'"

RecurbBow-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/Weapons/PrimalItem_WeaponRecurveBow.PrimalItem_WeaponRecurveBow'"

Dinghy (you might have it listed as sail boat?)-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/PrimalItemRowBoat.PrimalItemRowBoat'"
Oil Tank-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/PrimalItemStructure_OilTank.PrimalItemStructure_OilTank'"
Viking Axe-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/Weapons/PrimalItem_VikingAxe.PrimalItem_VikingAxe'"
Battle Axe-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/Weapons/PrimalItem_WeaponBattleAxe.PrimalItem_WeaponBattleAxe'"
Grindstone-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/PrimalItemStructure_Grindstone.PrimalItemStructure_Grindstone'"
Windmill-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/PrimalItemStructure_Windmill.PrimalItemStructure_Windmill'"
Hammock-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/PrimalItemStructure_Hammock_Simple.PrimalItemStructure_Hammock_Simple'"
Big Campfire-
"Blueprint'/Game/Mods/PrimitivePlusMod/Items/PrimalItemStructure_BigCampfire.PrimalItemStructure_BigCampfire'"

I think that's it. I can send you more as I find them if you like.

You can remove
"apiary"
"Bee- queen"
"Bee-drone"
"Bee-worker"
Spinning mule
Fabric
copper
and possibly latex(I have not seen any of these in game)

Here is a list of all prim plus engrams :)

https://ark.gamepedia.com/Engrams_(Primitive_Plus)

**Comment by thommcgrath** [comment]
Thanks, I've made most of those changes, though I'm not sure what to do about the Sailboat/Dinghy issue. I guess I should delete the sailboat. The dinghy and sailboat use different classes, so they are distinct objects, but the wiki maps them together. So I'm not sure the history of the item(s).

And for the record, I got my original list from that very page. Seems it wasn't up to date when I did that.
